class Main
{
    public static void main(String args[])
    {
        try
        {
            int d=5/2;
            System.out.println("Rest of code");
        }
        
        catch(ArithmeticException e)
        {
            System.out.println("Exception "+e.getMessage());
        }
        finally
        {
             System.out.println("This is finally block");
        }
    }
    
}
